### PR TITLE
Updated navigation & join metagame

### PR DESCRIPTION
**Pull request by peth-yursick**
This pull request updates:
- The navigation board post with a new link to the new board & removal of outdated info
- The join metagame page with removal of the typeform as decided on the innkeepers call & replacement of it with a newsletter subscription form

### CODE REVIEW

**Comment by peth-yursick** [comment]
halp plz @luxumbra @Baconaetor

**Comment by luxumbra** [comment]
> halp plz @luxumbra @Baconaetor

I didn't want to tread on Bacons toes with this but can take a look

**Review by luxumbra — CHANGES_REQUESTED** [review]
I have fixed the build but for some reason, the styling of the wiki sidebar is broken. Looking into it.

**Review by Baconaetor — COMMENTED** [review]
It's the point here because it's directing you to the sidebar as you're already on the page. Long term I think I just want to update this with the new icons like on MyMeta.

**Comment by Baconaetor** [comment]
> 
> 
> > halp plz @luxumbra @Baconaetor
> 
> I didn't want to tread on Bacons toes with this but can take a look

Lol you're good, I don't know why I keep missing these. Gotta figure out how to get better notifications somehow.

**Comment by Baconaetor** [comment]
> 
> 
> I have fixed the build but for some reason, the styling of the wiki sidebar is broken. Looking into it.

Yeah this is weird. What was wrong with the build to start?

**Comment by luxumbra** [comment]
> > I have fixed the build but for some reason, the styling of the wiki sidebar is broken. Looking into it.
> 
> Yeah this is weird. What was wrong with the build to start?

The sidebars file was throwing an error due to the 300 in the `300-of-metagame.md` file not gettng picked up and there was a style attribute in one of the iframes peth added which made the md file invalid.

**Comment by peth-yursick** [comment]
Thank you @luxumbra, seems to be working now?

File: src/css/custom.scss [hunk 1]
```diff
@@ -100,7 +100,7 @@ body,
     }
 }
 
-.main-wrapper [role='complementary'] {
+.main-wrapper [class^='docSidebarContainer'] {
```
Comment (luxumbra) [inline src/css/custom.scss:103]: This was the cause of breaking styling with the sidebar. Docusaurus is spitting our slightly different html & attributes (swapped `<div role="complementary">` for `<aside>`)

**Review by luxumbra — COMMENTED** [review]
Looks like this is fixed. @Baconaetor or @peth-yursick can you please verify that the breakage is now resolved. 🙏🏻 Then we can merge

**Review by luxumbra — APPROVED** [review]

**Comment by luxumbra** [comment]
> It's the point here because it's directing you to the sidebar as you're already on the page. Long term I think I just want to update this with the new icons like on MyMeta.

So you think 👈🏻  should stay? On the playbooks call the other day the general consensus was "why the f would you use a pointy finger?!" and to replace it with books. 😅

**Comment by Baconaetor** [comment]
> 
> 
> > It's the point here because it's directing you to the sidebar as you're already on the page. Long term I think I just want to update this with the new icons like on MyMeta.
> 
> So you think 👈🏻 should stay? On the playbooks call the other day the general consensus was "why the f would you use a pointy finger?!" and to replace it with books. 😅

Lol yeah, it's not great. I would have kept it for the time being as it prevents it from looking like a dead link. When you click it nothing happens, so the point helped direct you to the fact you're already there. That whole app drawer needs to change though. We need a new homepage for the wiki desperately. I just can't think of a good idea for one. . .

**Comment by luxumbra** [comment]
> > > It's the point here because it's directing you to the sidebar as you're already on the page. Long term I think I just want to update this with the new icons like on MyMeta.
> > 
> > 
> > So you think 👈🏻 should stay? On the playbooks call the other day the general consensus was "why the f would you use a pointy finger?!" and to replace it with books. sweat_smile
> 
> Lol yeah, it's not great. I would have kept it for the time being as it prevents it from looking like a dead link. When you click it nothing happens, so the point helped direct you to the fact you're already there. That whole app drawer needs to change though. We need a new homepage for the wiki desperately. I just can't think of a good idea for one. . .

Dammit lol it's been merged already. 🤦🏻  Hard to know how much work to put into it when we are meant to be migrating the wiki away from Docusaurus. Agreed though, the current landing page would be great. Wouldn't be too hard to improve on the current one.

**Comment by peth-yursick** [comment]
> so the point helped direct you to the fact you're already there. That whole app drawer needs to change though. We need a new homepage for the wiki desperately. I just can't think of a good idea for one. . .

Yep, that's precisely the reason I used it. The app drawer needs to go but I think the landing page for the wiki itself should simply be a nicer way to display all the categories. Maybe similar to how playbooks are displayed but wider?

image

This is a bit too wide but you get the idea.. Will need a designer on it.

MetaGame itself will have a new landing page & the wiki will be one of the things accessed through the menu above & then different pieces will I guess over time be taken out of the wiki (eg. Playbooks & Great Houses) to be put in their own place.

> Hard to know how much work to put into it when we are meant to be migrating the wiki away from Docusaurus.

For Playbooks yes, but didn't we agree for the wiki itself to only upgrade it to Docusaurus v2?

**Comment by Baconaetor** [comment]
> 
> 
> > so the point helped direct you to the fact you're already there. That whole app drawer needs to change though. We need a new homepage for the wiki desperately. I just can't think of a good idea for one. . .
> 
> Yep, that's precisely the reason I used it. The app drawer needs to go but I think the landing page for the wiki itself should simply be a nicer way to display all the categories. Maybe similar to how playbooks are displayed but wider?
> 
> image
> 
> This is a bit too wide but you get the idea.. Will need a designer on it.
> 
> MetaGame itself will have a new landing page & the wiki will be one of the things accessed through the menu above & then different pieces will I guess over time be taken out of the wiki (eg. Playbooks & Great Houses) to be put in their own place.
> 
> > Hard to know how much work to put into it when we are meant to be migrating the wiki away from Docusaurus.
> 
> For Playbooks yes, but didn't we agree for the wiki itself to only upgrade it to Docusaurus v2?

Oooh, I'm liking this. I can work with that. Also yeah, we need more clarity on the Docusaurus stuff. Are we still on V1 currently? Also, who's making that call?

**Pull request merged**